srcpkgs/seatd/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'seatd'
+pkgname=seatd
+version=0.4.0
+revision=1
+build_style=meson
+configure_args="-Dexamples=disabled $(vopt_if elogind -Dlogind=enabled)"
+hostmakedepends="pkg-config scdoc"
+makedepends="$(vopt_if elogind elogind-devel)"
+short_desc="Minimal seat management daemon"
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
+license="MIT"
+homepage="https://kl.wtf/projects/seatd/"
+distfiles="https://git.sr.ht/~kennylevinsen/seatd/archive/${version}.tar.gz"
+checksum=3c7458ff0c0d41ddd7734d01a5c5a9110ee5b94f01931e7e41ea0fd0c2e6050f
+system_groups=seatd
```

Comment:
The group should be prefixed by an underscore (`_seatd`) as [described in Manual.md](https://github.com/void-linux/void-packages/blob/master/Manual.md#creating-system-accountsgroups-at-runtimel)

Reply:
That applies only to accounts not groups, assuming I read the manual correctly.

Reply:
Generally applies to both as a type of namespace between system generated groups besides the base set of groups and user created groups.

Reply:
Seems reasonable, we should make the wording in the manual more clear

Reply:
#26230